scripts/install_deps.sh

```diff
@@ -17,6 +17,7 @@ apt-get install -y \
     libssl-dev \
     libtinfo-dev \
     libzstd-dev \
+    llvm-11-dev \
```

Comment:
This shouldn't be required

Reply:
OK, removing it.

Reply:
> This shouldn't be required

Only reason I added this is that when uou install `llvm-dev` on ubuntu, it installs llvm-10 by default.

Reply:
Zach had the same confusion around this script; it really ought to be called `install_deps_for_pinned_build.sh`, as that's its purpose. Notice how it doesn't install boost either.

Reply:
I see. llvm is installed in `scripts/pinned_build.sh`. Unfortunately it installs version `7.0.1`.